addons/custom-menu-bar/menu-labels.css

```
@@ -6,7 +6,7 @@
   justify-content: center;
 }
 
-[class*="menu-bar_hoverable"] > span {
+[class*="menu-bar_hoverable"] > span:not(.sa-editormessages-count) {
```

I think it's better to more precisely specify what the selector should match instead of excluding things it shouldn't.
```suggestion
[class*="settings-menu_dropdown-label_"],
[class*="menu-bar_collapsible-label_"] {
```